tinygrad/codegen/linearizer.py

```diff
@@ -356,15 +343,16 @@ def end_loop(xx:List[Variable]):
         # end the late reduce loop
         end_loop(end_local_idxs)
         self.load_cache.clear()
+        acc_idxs = global_idxs+local_idxs+fake_reduce_idxs+upcast_idxs
 
     # load latebufs
-    loaded_buffers.update({b:self.global_load(i, global_idxs+local_idxs+fake_reduce_idxs+upcast_idxs) for i,b in enumerate(self.bufs) if b not in self.earlybufs and i != 0 and b.__class__ is not LocalBuffer})
+    loaded_buffers.update({b:self.global_load(i, acc_idxs) for i,b in enumerate(self.bufs) if b not in self.earlybufs and i != 0 and b.__class__ is not LocalBuffer})
```

Comment:
Do you have to change this? I like the sum, makes it clearer what it's doing

Reply:
Because of the acc_idxs rewrites when it's a tensor core (lines 249-254), I think it needs to be this way (reverting these changes to linearizer breaks HIP).

Reply:
Err, yea, this is confusing. It's reassigned on line 344 also. There has to be a way to expose this better